src/vt/configs/arguments/args.cc

```diff
@@ -763,6 +1420,10 @@ std::tuple<int, std::string> ArgConfig::parseToConfig(
 
   app.set_help_flag("--vt_help", "Display help");
 
+  // Find a YAML config file, if present
+  addYamlConfig(app, appConfig);
+
+  // Parse remaining args
```

Comment:
Misleading comments: parsing happens later, this is just preparation:
```suggestion
  // setup CLI parsing
  addYamlConfig(app, appConfig);
```

Reply:
Fixed